app/views/followships/update.js.erb

```diff
@@ -0,0 +1,10 @@
+alert("here is")
```

Comment:
Please clean you code, before publishing

Reply:
Done